src/Tickets/Seating/app/blockEditor/store/index.js

```diff
@@ -104,7 +111,26 @@ const store = createReduxStore(storeName, {
 					},
 					seatTypesByPostId: {
 						...state.seatTypesByPostId,
-						[ticketPostId]: action.seatTypeId,
+						[ticketPostId || action.clientId]: action.seatTypeId,
+					},
+				};
+			case 'SET_TICKET_SEAT_TYPE_BY_POST_ID':
+				const ticketId = getTicketIdFromCommonStore(action.clientId);
+
+				const { seatTypesByPostId, seatTypesByClientId } = state;
+
+				const seatTypeId =
```

Comment:
I'm not sure about the lookup order here: shouldn't the ticket ID be looked up first, then the post, then the client ID? I.e. I _think_ the order should be the reverse of what it is now.

Reply:
I think the current approach is correct. Let me explain my reasoning.

1. If the seat type id is present in the `seatTypesByClientId` property this is the more recent since this gets updated in every change. 
2. Otherwise, we check `seatTypesByPostId` in the clientId offset since this is being set up along with the previous value. - making this unnecessary as a result
3. We check in `seatTypesByPostId` for when the value is not present `seatTypesByClientId` - e.g. during initial load of the block.